# Parasite-Project

Paper Title: Transcriptomic profiling of Alexandrium fundyense during physical interaction with or exposure to chemical signals from the parasite Amoebophrya

Authors: Lu et al. 2016

Project Overview: The ‘Amoebophrya species complex’ is part of the order Syndiniales, a diverse lineage of parasites known to infect dinoflagellates (including harmful algal bloom taxa), ciliates, radiolarians, and copepods. These parasites dominate molecular marker gene libraries from global ocean surveys; however, there is a widening gap between the characterization of the diversity of this order and the roles these microbes play in marine environments, including pressure on populations of susceptible hosts. Here, the authors used cultured isolates of the host, Alexandrium fundyense, and a strain of parasite belonging to the ‘Amoebophrya species complex’ to compare the effects of an active parasite presence and waterborne cues that may be released from infections in nearby cells on host gene expression. Three time points were collected based on the progression of an infection when parasites were present: time 0 when the hosts were exposed to parasite cues, the initial stage of infection at hour 6, and the termination of infections at hour 96. 

Folders:
jupyter-notebook/ contains all code related to the visualization of the differential gene expression and gene annotation data

data/
  -raw-sequences/ all raw sequencing files downloaded for this study
  -DE-analysis/ all files created for differential gene expression analyses
  
output/
  -alignment/ result of read alignment
  -DE-analysis/ compiled results of differential expression analysis
  -gene-annotation/ result of gene annotation of differentially expressed genes

Step 1: Download files from SRA
Thirty three single end read files are available publically from this study. To supplement the reads available from this study, a reference transcriptome from MMETP was also downloaded. 

##Location of data files

Step 2: Assemble reference transcriptome

##Location of data files and any relevant scripts

Step 3: Align reads to reference transcriptome

##Location of assembled transcriptome/relevant scripts


